tools/monorepo-utils/src/ci-jobs/lib/config.ts

```diff
@@ -21,6 +21,11 @@ export const enum JobType {
 	Test = 'test',
 }
 
+/**
+ * The type of the test job.
+ */
+export const testTypes = [ 'default', 'e2e', 'api', 'performance' ];
```

Comment:
```suggestion
export const testTypes = [ 'default', 'e2e', 'api', 'performance' ] as const;
```

If we use a `const` assertion here we can add type specificity to `testType` correctly.